rumbl/web/controllers/auth.ex

```diff
@@ -8,22 +8,33 @@ defmodule Rumbl.Auth do
 
   def call(conn, repo) do
     user_id = get_session(conn, :user_id)
+
     cond do
       user = conn.assigns[:current_user] -> conn
+        put_current_user(conn, user)
```

Comment:
@shouston3 both would be valid. This is one fewer line and achieves the objective.
As soon as we need to pipe more than one function we would convert it. 👍